schedule-2019-03-internal.md

```diff
@@ -0,0 +1,95 @@
+# Instructor Training (Internal, March 2019)
+
+Copyright 2019, RStudio Inc.
+All material made available under the [Creative Commons Attribution-NonCommercial 4.0](https://creativecommons.org/licenses/by-nc/4.0/) license.
+
+## Instructors
+
+- Garrett Grolemund
+- Greg Wilson
+
+## Day 1
+
+1.  Getting Started (0:15/09:15)
+
+1.  Two Ways to Teach (0:45/10:00)
```

Comment:
Can we begin with the active vs passive learning poll, and even my Bueller and Gandalf videos to set a fun, interactive tone?

Reply:
I've added active vs. passive, but playing videos for an online group is problematic - I'd rather leave that 'til the Nederbragt videos of bad and good teaching.